.photofinish.toml

```diff
@@ -444,4 +444,16 @@ files = [
 
 files = [
     "./test/fixtures/scenarios/healthy-27-node-SAP-cluster/9a3ec76a-dd4f-5013-9cf0-5eb4cf89898f_sap_system_discovery.json"
+]
+
+[demo]
```

Comment:
nitpick:
i would put this on top, as it is not "just yet another" fixture. It is important, so it doesn't get lost in between other new entries